.pre-commit-config.yaml

```diff
@@ -1,4 +1,9 @@
 repos:
+  - repo: https://github.com/myint/autoflake
+    rev: master
```

Comment:
We should never reference a branch in pre-commit hooks, otherwise it defeats the pre-commit caching mechanism. It must be a tag or commit.

Reply:
done, lasted version are used https://github.com/myint/autoflake/releases/tag/v1.4